src/components/admin/Billing/index.tsx

```diff
@@ -35,25 +42,43 @@ function AdminBilling() {
     const setHydrated = useBilling_setHydrated();
     const setHydrationErrorsExist = useBilling_setHydrationErrorsExist();
 
+    const selectedTenant = useBilling_selectedTenant();
     const setBillingHistory = useBilling_setBillingHistory();
+    const updateBillingHistory = useBilling_updateBillingHistory();
     const setDataByTaskGraphDetails = useBilling_setDataByTaskGraphDetails();
 
     const resetBillingState = useBilling_resetState();
 
+    const [historyInitialized, setHistoryInitialized] =
+        useState<boolean>(false);
+
+    const currentMonth = useConstant(() => {
+        const today = new Date();
+
+        return startOfMonth(today);
+    });
```

Comment:
Not for this PR... but we should look into creating a hook or something for all our date/time needs so they all reference a single copy given how often we are calculating this stuff now.